Dockerfile

```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get install nodejs -y
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 COPY . /usr/src/app
-RUN npm install -g npm && npm install -g ember-cli@latest && npm install -g bower
-RUN npm install
+RUN npm install -g ember-cli
+RUN latest && npm install -g bower
```

Comment:
The `latest` should have been on the previous line (eg  npm install -g ember-cli@latest)